### ISSUE TITLE
Documentation of function-like macro parameter names

### ISSUE THREAD

**Issue by nwhetsell** [csound/manual#100]
The documentation of `#define` at https://csound.github.io/docs/manual/define.html says of parameter names in function-like macros, “There may be up to 5 arguments, and the names may be any choice of letters.” This appears to be inaccurate. It seems that the number of parameters of a function-like macro has no limit (besides memory), and parameter names can begin with a letter or underscore and contain letters, underscores, and digits. Running this

```csound
#define MACRO(_arg1' _arg2' _arg3' _arg4' _arg5' _arg6' _arg7' _arg8)
  #$_arg1 $_arg2 $_arg3 $_arg4 $_arg5 $_arg6 $_arg7 $_arg8#
prints "==> $MACRO(1'2'3'4'5'6'7'8)\n"
scoreline_i "e"
```

prints

```
==> 1 2 3 4 5 6 7 8
```

**Comment by jpffitch** [comment]
manual canged

**Comment by nwhetsell** [comment]
This was addressed in commits 560419791cda46295d277bbc814738217bb6918f and 5ad352378d938d499abf782af1836cc127394a82, so closing.
